openmc/mesh.py

```diff
@@ -859,7 +905,7 @@ def calc_mesh_volumes(self):
             Volumes
 
         """
-
+        self._volume_dim_check()
         V_r = np.diff(np.asarray(self.r_grid)**2 / 2)
```

Comment:
Looking at the rest of the getters and setters for the grid properties on meshes, they don't appear to be consistent. Would you be able to add `np.asarray` to all the setters so that the return type of `r_grid`, `theta_grid`, `x_grid`, etc are `np.ndarray`?  I also can't make suggestions on collapse code not in this PR, but I think it makes sense to add those small fixes to this PR.

Reply:
Good call. I'd say that falls into the realm of "while we're here". 😄 Just made that adjustment.